### PR TITLE
Add sample token

### PR DESCRIPTION
**Pull request by amityadav0**

### CODE REVIEW

File: sample-token/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#################
+#   NEARswap    #
+
+
+include ../Makefile_common.mk
+
+export NCLP_ACC=beta-1.nearswap.testnet
```
Comment (robert-zaremba) [inline sample-token/Makefile:7]:
```suggestion
export TOKEN1=token1.nearswap.testnet
```

Comment (robert-zaremba) [reply sample-token/Makefile:7]: rest of the file should be updated accordingly.

**Review by robert-zaremba — APPROVED** [review]
pre-approving.

File: sample-token/Makefile (hunk 1 above)
Comment (amityadav0) [reply sample-token/Makefile:7]: ok

**Pull request merged**